gno.land/genesis/genesis_balances.txt

```diff
@@ -62,6 +62,7 @@ g1f977l6wxdh3qu60kzl75vx2wmzswu68l03r8su=10000000000ugnot # @catShaark #92
 g1644qje5rx6jsdqfkzmgnfcegx4dxkjh6rwqd69=10000000000ugnot # @MadafakAvril14th #91
 g1mzjajymvmtksdwh3wkrndwj6zls2awl9q83dh6=10000000000ugnot # @Vanlee #89
 g1qpymzwx4l4cy6cerdyajp9ksvjsf20rk5y9rtt=10000000000ugnot # @zooma
+g1exgus562z8j4cmxt4fvva4kmd0aktwptnes5ws=10000000000ugnot # @zkey
```

Comment:
You should uncommit this change from the `genesis_balances.txt`, as this file should not be modified at all

Reply:
Oops, that was a mistake, sorry about that.

Reply:
This modification has not been reverted

Reply:
Yes, slipped through the cracks. Here is the reversion: https://github.com/schollz/gno/commit/2f429b65028b42a3d69b50c0e05b06bffff6a915